cluster-autoscaler/core/autoscaling_context.go

```diff
@@ -90,9 +90,6 @@ type AutoscalingOptions struct {
 	CloudProviderName string
 	// NodeGroups is the list of node groups a.k.a autoscaling targets
 	NodeGroups []string
-	// VerifyUnschedulable is used to enable verification to ensure that each pod marked by Scheduler as unschedulable actually can't be scheduled on any node.
-	// This prevents from adding unnecessary nodes in situation when CA and Scheduler have different configuration.
-	VerifyUnschedulablePods bool
```

Comment:
Previously we allowed user to use this flag to disable FilterOutSchedulable check. However, with no filtering based on PodScheduled condition FilterOutSchedulable is required to prevent double scale-up for the same pod.

So with the change in logic setting this flag to false would basically break CA. Since there was no real benefit to changing it and I never heard about anyone doing it, I think it's much safer to remove it altogether.